src/materials/P91LAROMANCEStressUpdate.C

```diff
@@ -0,0 +1,5324 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "P91LAROMANCEStressUpdate.h"
+
+registerMooseObject("BlackBearApp", P91LAROMANCEStressUpdate);
+
+InputParameters
+P91LAROMANCEStressUpdate::validParams()
+{
+  InputParameters params = ADLAROMANCEStressUpdateBase::validParams();
+  params.addClassDescription("LAROMANCE creep update model for P91");
+
+  // Override defaults for material specific parameters below
+  params.addRangeCheckedParam<Real>("initial_cell_dislocation_density",
+                                    5.0e12,
+                                    "initial_cell_dislocation_density >=1.0+12 & "
+                                    "initial_cell_dislocation_density <= 6.0E+12",
+                                    "Initial density of cell (glissile) dislocations (1/m^2)");
+  params.addRangeCheckedParam<Real>(
+      "max_relative_cell_dislocation_increment",
+      0.5,
+      "max_relative_cell_dislocation_increment > 0.0",
+      "Maximum increment of density of cell (glissile) dislocations.");
+  params.addRangeCheckedParam<Real>(
+      "initial_wall_dislocation_density",
+      10.0e12,
+      "initial_wall_dislocation_density >= 6.0e12 & "
+      "initial_wall_dislocation_density <= 18.0e12",
+      "Cell wall (locked) dislocation density initial value (1/m^2).");
+  params.addRangeCheckedParam<Real>(
+      "max_relative_wall_dislocation_increment",
+      0.5,
+      "max_relative_wall_dislocation_increment > 0.0",
+      "Maximum increment of cell wall (locked) dislocation density initial value (1/m^2).");
+  return params;
+}
+
+P91LAROMANCEStressUpdate::P91LAROMANCEStressUpdate(const InputParameters & parameters)
+  : ADLAROMANCEStressUpdateBase(parameters)
+{
+}
+
+std::vector<unsigned int>
+P91LAROMANCEStressUpdate::getTilings()
+{
+  // indicies are [input]
+  return {1, 1, 3, 1, 3, 1};
+}
+
+std::vector<std::vector<std::vector<ROMInputTransform>>>
+P91LAROMANCEStressUpdate::getTransform()
+{
+  return {{{ROMInputTransform::LOG,
```

Comment:
This might be more readable and more consistent with the formatting of the return statement in the function below if we combined the 6 vector entries on each line. There are clang-format overrides to allow you to have longer lines.